src/main/java/com/openshift/restclient/model/build/IBinaryBuildSource.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+
+package com.openshift.restclient.model.build;
+
+public interface IBinaryBuildSource extends IBuildSource {
+
+    /**
+     * The asFile
+     * 
```

Comment:
I'd remove this javadoc as it offers no added value or, alternatively comment what this method does.